pkgs/applications/networking/cluster/marathon/default.nix

```diff
@@ -17,14 +17,14 @@ stdenv.mkDerivation rec {
 
     makeWrapper ${jdk.jre}/bin/java $out/bin/marathon \
       --add-flags "-Xmx512m -jar $out/libexec/marathon/${name}.jar" \
-      --prefix "MESOS_NATIVE_JAVA_LIBRARY" : "$MESOS_NATIVE_JAVA_LIBRARY"
```

Comment:
The Mesos package brings that env var into scope, so it's probably best to keep using it for the sake of cross platform comparability (OS X dlls end in .dylib, for example)